lib/updateFacebookCache.js

```diff
@@ -0,0 +1,15 @@
+var axios = require('axios');
+var config = require('../config/config.json');
+
+// update facebook's cache with most recent metadata
+function updateFacebookCache ( url ) {
+	axios.post('https://graph.facebook.com', { id: url, scrape: true, access_token: config.fbAccessToken })
+	  .then(function (response) {
+	    console.log(response);
+	  })
+	  .catch(function (error) {
+	    console.log(error);
+	  });
+}
+
```

Comment:
you'll need another function here with no arguments that queries all the urls that need to be updated, and then runs `updateFacebookCache(url)` for each one.

(and then we'll run that every hour -- that won't be in the code.)